crates/gitbutler-core/src/virtual_branches/remote.rs

```diff
@@ -129,12 +129,7 @@ pub fn branch_to_remote_branch(branch: &git::Branch) -> Result<Option<RemoteBran
                         None
                     },
                     name,
-                    last_commit_timestamp_ms: commit
-                        .time()
-                        .seconds()
-                        .try_into()
-                        .map(|t: u128| t * 1000)
-                        .ok(),
+                    last_commit_at: Some(commit.time()),
```

Comment:
It looks like the desired type of `last_commit_at` is `git2::Time`, then it would match naturally at least my concern about `Duration` would be addressed.